preflight-checks/src/utils/tracks.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import Analytics from './tracks/analytics';
+import Tracks from './tracks/analytics/clients/tracks';
+let analytics = null;
+
+async function init() {
+    const tracks = new Tracks( 0, 'anon', 'vip_preflight_' );
+    analytics = new Analytics( tracks );
+
+    return analytics;
+}
+
+async function getInstance() {
+    if ( analytics ) {
+        return analytics;
+    }
+
+    analytics = init();
+
+    return analytics;
+}
```

Comment:
unless this is particularly heavy, it's ok to create this in the outer scope, since it will always be needed

```suggestion
const tracks = new Tracks( 0, 'anon', 'vip_preflight_' );
const client = new Analytics( tracks );
```